src/ebonite/ext/sqlalchemy/models.py

```diff
@@ -153,6 +153,8 @@ def to_obj(self) -> Model:
                       params=safe_loads(self.params, Dict[str, Any]),
                       id=self.id,
                       task_id=self.task_id)
+        for image in self.images:
```

Comment:
same fix for image and env models?
hmmm, or we dont have .instances attribute in image and env? are they needed?

Reply:
also, in image `to_obj` for image variable is called `model` instead of `image`. This is the PR to fix it)

Reply:
Fixed naming in `SImage.to_obj`.

We don't have `instances` in `Image` and `RuntimeEnvironment` so there no need for the same fix. The motivation behind having no attribute here is that unlike `Project`-`Task`, `Task`-`Model` and `Model`-`Image` binary relationships we have a ternary one: `Image`-`RuntimeInstance`-`RuntimeEnvironment`. In my opinion adding this attribute could lead to quite complicated logic for adding and deleting. Moreover, we could expect that both `Image` and `RuntimeEnvironment` could have hundreds of `RuntimeInstance`s associated which could cause performance problems.